src/ui/components/Notification.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <chrono>
+#include <string>
+
+namespace DigitizerUi::components {
+
+struct Notification {
+    std::string               text;
+    std::chrono::milliseconds dismissTime{5000};
+
+    static void success(Notification&& notification);
+    static void success(const std::string& text, std::chrono::milliseconds dismissTime = std::chrono::milliseconds{5000});
+
+    static void warning(Notification&& notification);
+    static void warning(const std::string& text, std::chrono::milliseconds dismissTime = std::chrono::milliseconds{5000});
+
+    static void error(Notification&& notification);
+    static void error(const std::string& text, std::chrono::milliseconds dismissTime = std::chrono::milliseconds{5000});
```

Comment:
IMO (non-recoverable) errors should have a longer dismissTime since they should happen quite rarely and are probably not something you can quickly retry.

Reply:
ok, maybe we should also add a permanent option (ie require closing)